_posts/2025-03-13-CPANSec-Retrospective-2024.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: post
+title:  "CPANSec retrospective 2024"
+date:   2025-03-13 00:23:17 +0200
+categories: cpansec update
+author: Thibault Duponchelle
+excerpt: Here is the CPANSec 2024 Retrospective
+---
+
+That was a big year for CPANSec. 
+
+If it gets a bit under the radar to you, please find some of the achievements listed in this retrospective!
+
+## Common Vulnerabilities and Exposures (CVEs)
+This year was the year where CPANSec installed a new mindset for Perl 
+and their relationship towards Common Vulnerabilities and Exposures (CVEs).
+
+Until then, very few CVEs were emitted for Perl modules and tooling. 
+And people and authors were a bit reluctant to deal with that process.
+
+CVE are not a sign of bad quality or maintenance of a code, 
+but a professionnal way of tagging and tracking a vulnerability, 
+also proving the importance of the concerned code.
+
+### Vulnerable Spreadsheet Parsing modules
+Start of the year was marked by vulnerabilities reported for [Spreadsheet::ParseExcel](https://metacpan.org/pod/Spreadsheet::ParseExcel).
+
+- [CVE-2023-7101: Spreadsheet::ParseExcel arbitrary code execution vulnerability](https://security.metacpan.org/2024/02/10/vulnerable-spreadsheet-parsing-modules.html#cve-2023-7101-spreadsheetparseexcel-arbitrary-code-execution-vulnerability)
+- [CVE-2024-22368: Spreadsheet::ParseXLSX denial of service vulnerability](https://security.metacpan.org/2024/02/10/vulnerable-spreadsheet-parsing-modules.html#cve-2024-22368-spreadsheetparsexlsx-denial-of-service-vulnerability)
+- [CVE-2024-23525: Spreadsheet::ParseXLSX XML external entity attack vulnerability](https://security.metacpan.org/2024/02/10/vulnerable-spreadsheet-parsing-modules.html#cve-2024-23525-spreadsheetparsexlsx-xml-external-entity-attack-vulnerability)
+
+### Insecure modes of CPAN installers
+CPANSec continued with CPAN installer [cpanminus]() (a major installer of the Perl ecosystem)
```

Comment:
Maybe link to https://github.com/miyagawa/cpanminus or the metacpan page? (I guess the metacpan page is best?)

Reply:
Oh yes of course that was the plan